test/e2e/framework/framework.go

```diff
@@ -569,7 +569,7 @@ func (f *Framework) CreateServiceForSimpleApp(contPort, svcPort int, appName str
 // CreatePodsPerNodeForSimpleApp creates pods w/ labels.  Useful for tests which make a bunch of pods w/o any networking.
 func (f *Framework) CreatePodsPerNodeForSimpleApp(appName string, podSpec func(n v1.Node) v1.PodSpec, maxCount int) map[string]string {
 	nodes := GetReadySchedulableNodesOrDie(f.ClientSet)
-	labels := map[string]string{
+	labelmaps := map[string]string{
```

Comment:
nit: It's only one map, not "maps", `podLabels` would be more descriptive
```suggestion
	podLabels := map[string]string{
```